[2024-12-20T10:37:57.191+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-20T10:37:57.209+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sql_operator_dag.SQL_provider_task scheduled__2024-01-12T02:00:00+00:00 [queued]>
[2024-12-20T10:37:57.225+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sql_operator_dag.SQL_provider_task scheduled__2024-01-12T02:00:00+00:00 [queued]>
[2024-12-20T10:37:57.226+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-20T10:37:57.307+0000] {taskinstance.py:2889} INFO - Executing <Task(SQLExecuteQueryOperator): SQL_provider_task> on 2024-01-12 02:00:00+00:00
[2024-12-20T10:37:57.317+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=14979) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-20T10:37:57.320+0000] {standard_task_runner.py:72} INFO - Started process 14988 to run task
[2024-12-20T10:37:57.323+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'sql_operator_dag', 'SQL_provider_task', 'scheduled__2024-01-12T02:00:00+00:00', '--job-id', '5214', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgresql.py', '--cfg-path', '/tmp/tmpvf56dq3i']
[2024-12-20T10:37:57.327+0000] {standard_task_runner.py:105} INFO - Job 5214: Subtask SQL_provider_task
[2024-12-20T10:37:57.410+0000] {task_command.py:467} INFO - Running <TaskInstance: sql_operator_dag.SQL_provider_task scheduled__2024-01-12T02:00:00+00:00 [running]> on host a998b5b3d359
[2024-12-20T10:37:57.540+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mateo' AIRFLOW_CTX_DAG_ID='sql_operator_dag' AIRFLOW_CTX_TASK_ID='SQL_provider_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-12T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-12T02:00:00+00:00'
[2024-12-20T10:37:57.542+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-20T10:37:57.581+0000] {sql.py:278} INFO - Executing: 
        create table if not exists dag_runs(
            dt date,
            dag_id varchar,
            primary key (dt,dag_id) 
        )
        
[2024-12-20T10:37:57.592+0000] {base.py:84} INFO - Retrieving connection 'sql_operator_practice'
[2024-12-20T10:37:57.608+0000] {base.py:84} INFO - Retrieving connection 'sql_operator_practice'
[2024-12-20T10:37:57.624+0000] {sql.py:553} INFO - Running statement: 
        create table if not exists dag_runs(
            dt date,
            dag_id varchar,
            primary key (dt,dag_id) 
        )
        , parameters: None
[2024-12-20T10:37:57.644+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-20T10:37:57.651+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=sql_operator_dag, task_id=SQL_provider_task, run_id=scheduled__2024-01-12T02:00:00+00:00, execution_date=20240112T020000, start_date=20241220T103757, end_date=20241220T103757
[2024-12-20T10:37:57.709+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-20T10:37:57.722+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
